src/scarlet2/observation.py

```diff
@@ -25,8 +25,12 @@ class Observation(Module):
     """Metadata to describe what view of the sky `data` amounts to"""
     renderer: (Renderer, eqx.nn.Sequential) = eqx.field(static=True)
     """Renderer to translate from the model frame the observation frame"""
+    check_observation: bool = eqx.field(static=True, default=False)
+    """Whether to validation checks on the observation object. Default is False."""
```

Comment:
I don't think this needs to be a class attribute. It's called at the end of __init__, so no need to remember it past that point.

Reply:
Yep, you're right, I can remove this.

Reply:
Removed